telegram/ext/_contexttypes.py

```diff
@@ -33,7 +33,13 @@ class ContextTypes(Generic[CCT, UD, CD, BD]):
     Convenience class to gather customizable types of the :class:`telegram.ext.CallbackContext`
     interface.
 
-    .. seealso:: :any:`ContextTypes Example <examples.contexttypesbot>`
+    .. seealso:: `Architecture Overview <https://github.com/\
+        python-telegram-bot/python-telegram-bot/wiki/Architecture>`_,
+        `Storing Bot, User and Chat Related Data <https://github.com/\
+        python-telegram-bot/python-telegram-bot/wiki/Storing-bot%2C-user-and-chat-related-data>`_,
+        `Job Queue <https://github.com/python-telegram-bot/
```

Comment:
the effect on `JobQueue` of this class is rather small … let's skip it :)

Reply:
deleting link to Job Queue wiki page